### PR TITLE
Adding translators

### PR DESCRIPTION
**Pull request by UlionTse**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by UlionTse** [comment]
@conda-forge/core @jezdez @grayskull Sorry, I don't know what happend. My recipe was generated from `grayskull pypi --strict-conda-forge translators`.

**Comment by xhochy** [comment]
Please don't overwrite the example

**Comment by UlionTse** [comment]
@xhochy Ok, thanks.

**Review by xhochy — COMMENTED** [review]
Please remove the changed in the example recipe and then this should be good to go

**Comment by UlionTse** [comment]
@xhochy Thanks for helping, this is ready to review.

**Comment by UlionTse** [comment]
@conda-forge/help-python This is ready to review.

File: recipes/pyexecjs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+about:
+  home: https://github.com/doloopwhile/PyExecJS
+  summary: Run JavaScript code from Python
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - UlionTse
+    # - doloopwhile
```
Comment (xhochy) [inline recipes/pyexecjs/meta.yaml:42]: Don't add other people, not even as a comment, please.
```suggestion
```

Comment (UlionTse) [reply recipes/pyexecjs/meta.yaml]: OK

**Pull request merged**